CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## 7.0.1 (Next)
+## 7.1.0 (Next)
```

Comment:
Increment this in https://github.com/mongoid/mongoid-slug/blob/master/lib/mongoid/slug/version.rb as well please.